Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -1031,6 +1041,19 @@ YUI.add('ez-platformuiapp', function (Y) {
                 readOnly: true,
                 value: {}
             },
+
+            /**
+             * Default language code, as defined by the current siteaccess language config.
+             *
+             * @attribute defaultLanguageCode
+             * @default "eng-GB"
+             * @type {Object}
+             * @readOnly
+             */
+            defaultLanguageCode: {
```

Comment:
this name can be confusing, since it's only for content creation, this should appear somehow in the name (`contentCreationDefaultLanguageCode` ?)